docs/cms/developers/e2e-testing/index.md

```diff
@@ -0,0 +1,21 @@
+# End-to-end testing
+
+## Introduction
+
+**End-to-end testing (E2E testing)** checks if the software is working correctly from the end user point of view. It verifies all the flows and actions which the user may perform when using the application. E2E tests should be executed in a real-world environment like a web browser to ensure that the testing closely resembles the actual user experience.
+
+## Test content
+
+To run tests in the context of CMS we need to decide what `content` created by authors we will use. In Websight CMS for testing purposes we can use the content from the following folders:
```

Comment:
I'm not sure if we should write that: "In Websight CMS for testing purposes we can use the content from the following folders" - content placed in such modules is related only to the project structure that we choose. It's not enforced by WebSight itself. 
Maybe we should put some link to the starter project and we could write all this page in the starter context. 
I mean, we could write that we can write e2e test for our project and in the sample Starter project we have functional and visual tests. For functional test purposes, we created content in the /tests/content module.... And that we prepared visual tests for the /content to ensure that our initial content won't break....

Reply:
done